style.css

```diff
@@ -0,0 +1,81 @@
+:root{
+    --main-radius: 5px;
+    --main-padding: 5px;
+}
+
```

Comment:
add css reset after root so it will remove scrolling from your grid

```js
*{
padding:0;
    margin:0;
    box-sizing: border-box;
}
```

Reply:
done

Reply:
Have you commited changes @Adishaikh  ?